src/olympia/git/tests/test_utils.py

```diff
@@ -99,9 +99,6 @@ def apply_changes(repo, version, contents, path, delete=False):
 def test_temporary_worktree(settings):
     repo = AddonGitRepository(1)
 
-    output = _run_process('git worktree list', repo)
-    assert output.startswith(repo.git_repository.path)
```

Comment:
mmm, don't we want this to be correct?  Does it work _after_ the extraction instead?

Reply:
There are other tests ensuring add-ons are extracted in the right place, this one is just about `TemporaryWorktree`, so :shrug: . Not sure what we gain by testing the output of `git worktree list` here compared to a few lines below.

Reply:
but we don't test `repo.git_repository.path` is in the output further down, just the name